scenarioo-client/app/step/screenAnnotations/screenAnnotationsButton.html

```diff
@@ -1,5 +1,5 @@
 <!-- View Screen Annotations toggle button -->
-<div style="display:inline-block;" id="sc-showHideScreenAnnotationsButton" ng-click="screenAnnotationsButton.toggleAnnotationsVisible()" ng-show="screenAnnotationsButton.isAnnotationsButtonVisible">
+<div style="display:inline-block;" id="sc-showHideScreenAnnotationsButton" ng-click="screenAnnotationsButton.toggleAnnotationsVisible()" ng-show="screenAnnotationsButton.isAnnotationsButtonVisible()">
```

Comment:
there was a `()` missing - without `()` it will only return the function without calling the function - which in this case is again an expression that is almist always true as long as this function is defined.